docs/source/developer_zone/dev_environment.rst

```diff
@@ -127,7 +127,7 @@ Install development dependencies
 
 .. code:: bash
 
-    micromamba create -n mamba -c conda-forge -f dev/environment.yml
+    micromamba create -n mamba -c conda-forge -f dev/environment-dev-extra.yml
```

Comment:
I think `dev/environment-dev.yml` is more appropriate here as it's the basic mandatory environment, and then we can say that when using the provided CMake Preset, we may want to add `dev/environment-dev-extra.yml` as well.